test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -151,7 +151,7 @@ def setUp(self):
                               'use -p moms=<mom1>:<mom2>')
         self.serverA = self.servers.values()[0].name
         self.paths = self.get_paths()
-        if not self.paths:
+        if not self.paths['cpuset']:
```

Comment:
I suggest you change the skipTest message in this case to say "cpuset cgroup not mounted".

Reply:
Thanks Mike. Updated.